lib/slack/bot.ex

```diff
@@ -110,26 +110,24 @@ defmodule Slack.Bot do
 
   @doc false
   def websocket_handle({:text, message}, _conn, %{slack: slack, process_state: process_state, bot_handler: bot_handler} = state) do
-    message = prepare_message message
+    case prepare_message message do
```

Comment:
I was a little unsure of these changes, but they seem to jive with current master.